Directory.Build.targets

```diff
@@ -4,10 +4,6 @@
     <UseWpf>true</UseWpf>
     <UseWindowsForms>true</UseWindowsForms>
   </PropertyGroup>
-
-  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
```

Comment:
@clairernovotny was this a special build related to some signature stuff?
I have removed the `netcoreapp3.1` target for now.

Reply:
We need the .NET Core App 3.1 version as other things are consuming the Core nuget package for symbol validation. The Core version is using a private build of NuGet that does the signature validation.